.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: android
+jdk:
+- oraclejdk8
+android:
+  components:
+  - tools
+  - tools
+  - platform-tools
+  - build-tools-24.0.2
+  - android-24
+  - extra-android-m2repository
+  licenses:
+    - 'android-sdk-preview-license-52d11cd2'
+    - 'android-sdk-license-.+'
+script:
+- "./gradlew clean build --stacktrace"
```

Comment:
`clean` is kinda redundant at Travis because it always clones the repo from scratch. 

Also you can have `./gradlew check` to run unit tests, no?